lib/git.js

```diff
@@ -148,9 +155,21 @@ async function http_github_delete_auto_build(pg_pool, req, res, regex) {
     throw new common.NotFoundError('An auto build was not found for this app.');
   }
   let token = common.decrypt_token(config.encrypt_key, query_data[0].token);
-  await remove_webhook_if_needed(`${app.app_name}-${app.space_name}`, query_data[0].repo, token)
+  let response_message = '{"status":"successful"}';
+  try {
+    await remove_webhook_if_needed(`${app.app_name}-${app.space_name}`, query_data[0].repo, token)
+  } catch (err) {
+    if (err.code === 401) {
+      // 401 means the cached token was invalidated
+      response_message = `{"status":"error", "message":"Auto build removed, but the cached Github token was invalid- please manually remove any Akkeris hooks from the ${query_data[0].repo} repository"}`
```

Comment:
Can you throw a 
```
common.UnprocessibleEntityError(`Auto build removed, but the cached Github token was invalid- please manually remove any Akkeris hooks from the ${query_data[0].repo} repository`)
```
 instead of returning back a  successful 200? The 422 error should be displayed on the CLI or UI, but not indicate an error on our side (422 Unprocessible Entity basically means a business logic problem occurred)

Reply:
Definitely! ty